primitives/src/evm.rs

```diff
@@ -28,7 +29,7 @@ use frame_support::{
 use pallet_evm::{AddressMapping, HashedAddressMapping, OnChargeEVMTransaction};
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
-use sp_core::{Hasher, H160, H256, U256};
+pub use sp_core::{Hasher, H160, H256, U256};
```

Comment:
Is it necessary to make it `pub` ?

Reply:
not necessary, just make it easy to import H160 type.
Earlier we used `ethereum_types::H160` in some places but with new bump `primtive_types` version, it point to `sp_core::H160` directly, thus save us an extra import

I made it pub, astar-primtives is imported every where in codebase and easier to just import from it